roles/kube-addons/templates/signalfx.yml.j2

```diff
@@ -0,0 +1,25 @@
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: kube-signalfx
+  namespace: kube-system
+spec:
+  replicas: 1
+  selector:
+    app: kube-signalfx
+  template:
+    metadata:
+      labels:
+        app: kube-signalfx
+    spec:
+      containers:
+      - name: signalfx-agent
+        image: quay.io/signalfx/cadvisor-integration:latest
```

Comment:
I wanted to ask that we pin to a versioned tag, but I was disappointed to learn that this repo does not actually publish versioned images. This is fine.